help/view-a-messages-edit-history.md

```diff
@@ -25,6 +25,11 @@ Organization administrators can
     never see a **MOVED** label; messages that have been moved will be
     marked as **EDITED** just like those whose content has been edited.
 
+!!! keyboard_tip ""
+
+    You can also select a message, and use <kbd>Shift</kbd> + <kbd>H</kbd> to
+    view its edit history.
```

Comment:
Maybe something more like the keyboard tips in [emoji-reactions](https://zulip.com/help/emoji-reactions)?

> You can view the edit history of the selected message by using the <kbd>Shift</kbd> + <kbd>H</kbd> shortcut.